webapp/src/serial.tsx

```diff
@@ -136,13 +140,20 @@ export class Editor extends srceditor.Editor {
             const nvalue = parseFloat(m[3]);
             if (!isNaN(nvalue)) {
                 this.appendGraphEntry(niceSource, variable, nvalue, receivedTime)
-                return;
             }
         }
 
         this.appendConsoleEntry(data)
     }
 
+    appendRawData(data: string) {
+       this.rawDataBuffer += data;
+       let excessChars = this.rawDataBuffer.length - this.maxBufferLength;
```

Comment:
trim it by 30% to avoid constant trimming

Reply:
Thanks, done